### DISCUSSION TITLE
First images for a new e107 admin theme...

### DISCUSSION THREAD

**Discussion by Alex-e107nl** [e107inc/e107#5371] [General]
Because it is nice to have an example I used the possibilities of tabler, this way you immediately have a nice structure and the conversion is easy to do, it is just time that e107 is also updated to bootstrap 5 as an admin! ( [https://tabler.io/admin-template/preview](https://tabler.io/admin-template/preview) )
![scr-admin5-1](https://github.com/user-attachments/assets/40dca953-8918-4fd2-a87a-33a0ef1e02ed)
![scr-admin5-2](https://github.com/user-attachments/assets/dfbb6497-12aa-43b2-9782-011ecefe09fb)
![scr-admin5-3](https://github.com/user-attachments/assets/f810e6a8-a6f8-4245-8cd0-5593ff7d536f)

**Comment by Alex-e107nl** [comment]
And of course the front.. simple cerulean or whatever bootswatch, dark light and button-to-top in it..
![scr-admin5-4](https://github.com/user-attachments/assets/df818eac-c0a5-4261-8560-10df70431d33)

**Comment by Jimmi08** [comment]
@Alex-e107nl Perfect work.
Just idea. Could light/dark mode be used for 'IMODE' value? I mean, when you have a plugin and the path to images is set by IMODE value. 
I hit on this now and I have no idea how to solve it correctly with plugin only.  They were called generic images before.

**Reply by Alex-e107nl** [reply]
Thx @Jimmi08 I use the function from bootstrap with the colormode.js, they are just buttons with an attribute, if you do it once it stays that way until you choose another one.. IMODE doesn't mean anything to me at the moment but as soon as I put it in git you can see if we can add something of course! Oh I didn't place any images in DARK mode...

**Comment by Alex-e107nl** [comment]
Dark version....

![scr-admin5-5](https://github.com/user-attachments/assets/adc4fae6-31ba-4d37-9cb2-bfabf56a5d2d)

**Reply by Alex-e107nl** [reply]
Still some things to do, white text.... brrrrrr :-)
